### PR TITLE
implement 1st year/02 Roots of quadratic equation python solution

### PR DESCRIPTION
**Pull request by VickyGuo0907**
### Have you read the [Contributing Guidelines on Pull Requests]()?

- [X] Yes

### Description

(Write your answer here.)

### Checklist

- [X] I've read the contribution guidelines.
- [X] This PR meets all the requirements like Clean Code, proper README and Documentation.

### Related Issues or Pull Requests

Fixes: #5 

### Screenshots (if any)

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot** | **updated screenshot**|

### CODE REVIEW

File: 1st Year/02. Roots of a quadratic equation/quadratic.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+if __name__ == "__main__":
+
+  a, b, c = [int(x) for x in input("Enter the coefficients of x:").split()]
+
+  equationroots(a, b, c)
+
+  ## three test case
+  print("some test cases:")
```
Comment (atarax665) [inline 1st Year/02. Roots of a quadratic equation/quadratic.py:46]: No need to add this sample test cases.

File: 1st Year/02. Roots of a quadratic equation/quadratic.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  sql_val = math.sqrt(abs(dis))
+
+  if dis > 0:
+    r1 = (-b + sql_val)/(2 * a)
+    r2 = (-b - sql_val)/(2 * a)
+    print("Roots are real and distinct:")
+    print("Root 1: {} Root 2: {}".format(r1, r2))
+  ## root are real
+  elif dis == 0:
+    r1 = r2 = -b / (2 * a)
```
Comment (atarax665) [inline 1st Year/02. Roots of a quadratic equation/quadratic.py:27]:
```suggestion
    r1 = r2 = - b / (2 * a)
```

File: 1st Year/02. Roots of a quadratic equation/quadratic.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    print("Root is: {}".format(-c/b))
+    return
+
+  # calculating discriminate using formula
+  dis = b * b - 4 * a * c
+  sql_val = math.sqrt(abs(dis))
+
+  if dis > 0:
+    r1 = (-b + sql_val)/(2 * a)
+    r2 = (-b - sql_val)/(2 * a)
```
Comment (atarax665) [inline 1st Year/02. Roots of a quadratic equation/quadratic.py:22]:
```suggestion
    r2 = (- b - sql_val) / (2 * a)
```

File: 1st Year/02. Roots of a quadratic equation/quadratic.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    print("Liner Equation:")
+    print("Root is: {}".format(-c/b))
+    return
+
+  # calculating discriminate using formula
+  dis = b * b - 4 * a * c
+  sql_val = math.sqrt(abs(dis))
+
+  if dis > 0:
+    r1 = (-b + sql_val)/(2 * a)
```
Comment (atarax665) [inline 1st Year/02. Roots of a quadratic equation/quadratic.py:21]:
```suggestion
    r1 = (- b + sql_val) / (2 * a)
```

File: 1st Year/02. Roots of a quadratic equation/quadratic.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    print("Roots are real and distinct:")
+    print("Root 1: {} Root 2: {}".format(r1, r2))
+  ## root are real
+  elif dis == 0:
+    r1 = r2 = -b / (2 * a)
+    print("The roots are real and equal:")
+    print("Root 1: {} Root 2: {}".format(r1, r2))
+  # when discriminant is less than 0
+  else:
+    r = -b/(2 * a)
```
Comment (atarax665) [inline 1st Year/02. Roots of a quadratic equation/quadratic.py:32]:
```suggestion
    r = - b / (2 * a)
```

File: 1st Year/02. Roots of a quadratic equation/quadratic.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+  equationroots(a, b, c)
+
+  ## three test case
+  print("some test cases:")
+  list = [(1, 1, 1), (1, 7, 12), (1, -2, 1)]
+  for item in list:
+    print("a={} b={} c={}".format(item[0], item[1], item[2]))
+    equationroots(item[0], item[1], item[2])
+
```
Comment (atarax665) [inline 1st Year/02. Roots of a quadratic equation/quadratic.py:51]: Remove these extra line breaks.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: 1st Year/02. Roots of a quadratic equation/quadratic.py (hunk 1 above)
Comment (VickyGuo0907) [reply 1st Year/02. Roots of a quadratic equation/quadratic.py]: removed

File: 1st Year/02. Roots of a quadratic equation/quadratic.py (hunk 2 above)
Comment (VickyGuo0907) [reply 1st Year/02. Roots of a quadratic equation/quadratic.py]: resolved

File: 1st Year/02. Roots of a quadratic equation/quadratic.py (hunk 4 above)
Comment (VickyGuo0907) [reply 1st Year/02. Roots of a quadratic equation/quadratic.py]: resoved

File: 1st Year/02. Roots of a quadratic equation/quadratic.py (hunk 6 above)
Comment (VickyGuo0907) [reply 1st Year/02. Roots of a quadratic equation/quadratic.py]: solved

**Review by atarax665 — APPROVED** [review]
:+1: :+1:

**Pull request merged**